forwarding/src/main/kotlin/com/pkware/file/forwarding/ForwardingPath.kt

```diff
@@ -0,0 +1,74 @@
+package com.pkware.file.forwarding
```

Comment:
I originally moved forwarding into its own module because it shouldn't have been required when publishing, but for some reason it's required in order to integrate things into core-libraries

Reply:
i am moving this into its own package within file-attribute-caching